cmd/admin-profile-start.go

```diff
@@ -25,20 +25,20 @@ import (
 	"github.com/minio/minio/pkg/madmin"
 )
 
-var adminProfilingStartFlags = []cli.Flag{
+var adminProfileStartFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "type",
 		Usage: "Profiler type, possible values are 'cpu', 'mem', 'block', 'mutex' and 'trace'",
 		Value: "mem",
 	},
 }
 
-var adminProfilingStartCmd = cli.Command{
+var adminProfileStartCmd = cli.Command{
 	Name:            "start",
-	Usage:           "Start recording profiling data",
-	Action:          mainAdminProfilingStart,
+	Usage:           "Start recording profile data",
```

Comment:
In previous `Usage:`s, everything was in lower case...
`Usage:  "Start recording profile data",` => `Usage:  "start recording profile data",`